toolkit/tools/internal/packagerepo/repocloner/rpmrepocloner/rpmrepocloner.go

```diff
@@ -23,17 +24,26 @@ import (
 )
 
 const (
-	allRepoIDs             = "*"
-	builtRepoID            = "local-repo"
-	toolchainRepoId        = "toolchain-repo"
-	cacheRepoID            = "upstream-cache-repo"
-	squashChrootRunErrors  = false
-	chrootDownloadDir      = "/outputrpms"
-	leaveChrootFilesOnDisk = false
-	updateRepoID           = "mariner-official-update"
-	previewRepoID          = "mariner-preview"
-	fetcherRepoID          = "fetcher-cloned-repo"
-	cacheRepoDir           = "/upstream-cached-rpms"
+	RepoFlagMarinerDefaults = uint64(1) << iota
```

Comment:
I don't think we use bit flags very much, could we add a quick comment here for future readers?
```suggestion
	// Use bit shifted iotas to generate a set of bit flags (ie 0x001, 0x010, 0x100, ...)
	RepoFlagMarinerDefaults = uint64(1) << iota
```

Reply:
Could you also add a comment for each one inidcating its intended use?